include/openssl/bio.h

```diff
@@ -701,6 +701,23 @@ OPENSSL_EXPORT int BIO_do_connect(BIO *bio);
 #endif  // !OPENSSL_NO_SOCK
 
 
+// Message digest BIOs
+
+// BIO_f_md provides a filter |BIO| that digests any data passed through it.
+// The BIO must be initialized with |BIO_set_md| or |BIO_get_md_ctx| before it
+// can be used.
+OPENSSL_EXPORT const BIO_METHOD *BIO_f_md(void);
+
+// BIO_get_md_ctx writes a reference of |b|'s EVP_MD_CTX* to |*ctx|.
+OPENSSL_EXPORT int BIO_get_md_ctx(BIO *b, EVP_MD_CTX **ctx);
+
+// BIO_set_md set's |b|'s EVP_MD* to |md|.
+OPENSSL_EXPORT int BIO_set_md(BIO *b, const EVP_MD *md);
+
+// BIO_set_md set's |*md| to |b|'s |EVP_MD*|.
```

Comment:
doc comment check is [failing](https://us-west-2.codebuild.aws.amazon.com/project/eyJlbmNyeXB0ZWREYXRhIjoiWk5IUWJGRGxBcTJ2Mkp4WGF3dnBwYjc5V0ZZYSt5SVVGbkwvODkydTNTaVQ2V2FMN3hwa0tjSWNFemw2QWtCWW5welFWV3lpRFpKVitwejgvelhpRWh3NDNqcWhKalpPYW9hL2tLMDlJSDFPT1NkNyIsIml2UGFyYW1ldGVyU3BlYyI6Ilp4VXRNQXFGM1BZYVlaRkIiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D/build/f4857309-3395-4199-a82f-1826e5aa2219):

```
[Container] 2025/06/30 23:34:36.355426 Running command (cd util && go run ./doc.go)
Failed to generate output: while parsing include/openssl/bio.h: comment for "BIO_get_md" doesn't seem to match line include/openssl/bio.h:718
```

```suggestion
// BIO_get_md set's |*md| to |b|'s |EVP_MD*|.
```

Reply:
Ahhh thanks for the heads up